dev/io.openliberty.checkpoint_fat_mp/bnd.bnd

```diff
@@ -0,0 +1,213 @@
+#*******************************************************************************
+# Copyright (c) 2022, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src,\
+	test-applications/app1/src,\
+	test-applications/app2/src,\
+	test-applications/ejbapp1/src,\
+	test-applications/inventory/src/java,\
+	test-applications/system/src/java,\
+	test-applications/startupcode/src,\
+	test-bundles/test.checkpoint.config.bundle/src,\
+	test-applications/timeoutTest/src,\
+	test-applications/restClient/src,\
+	test-applications/jndiTest/src,\
+	test-applications/appsecurity/src,\
+	test-applications/webSocketTest/src,\
+	test-applications/eventMgr/frontendUI/src/java,\
+	test-applications/eventMgr/backendServices/src/java,\
+	test-applications/openAPItest/src,\
+	test-applications/mpjwt/src,\
+	test-applications/EJBearApp/src,\
+	test-applications/ServletEARapp/src,\
+	test-applications/webEARapp/src,\
+	test-applications/mphealth/src,\
+	test-applications/facesApp/src,\
+	test-applications/jsonb/src,\
+	test-applications/jsonp/src,\
+	test-applications/managedBeans/src,\
+	test-applications/mpTelemetry/src,\
+	test-bundles/test.checkpoint.bells.bundle/src,\
+	test-applications/bells/src,\
+	test-applications/virtualhost/src,\
+	test-applications/DefaultPrincipalMappingApp/src,\
+	test-applications/JSPWithEL/src,\
+	test-applications/concurrentApp/src,\
+	test-applications/mapCache/src,\
+	test-applications/webCacheApp/src,\
+	test-applications/xmlBindings/src,\
+	test-bundles/test-mbeans/src,\
+	test-applications/jaxrspropagation/src,\
+	test-applications/mpOpenAPIConfig/src
+
+fat.project: true
+
+fat.test.container.images: kyleaure/db2-ssl:3.0
+
+# Define additional tested features that are NOT present in any XML files in this bucket.
+# In this case, osgiConsole-1.0 is added programmatically at runtime.
+tested.features: \
```

Comment:
We should clean up the `tested.features` of this new FAT bucket as well as the original `io.openliberty.checkpoint_fat` bucket to only list the required features.  I think if you start with an empty list and build locally it should fail and tell you what list you need to add here.